helm/cluster-api-provider-azure/templates/admissionregistration.k8s.io_v1_validatingwebhookconfiguration_capz-validating-webhook-configuration.yaml

```diff
@@ -146,6 +146,7 @@ webhooks:
     apiVersions:
     - v1beta1
     operations:
+    - CREATE
```

Comment:
will this get reverted if we run `make generate` ?

Reply:
This got added when running `make generate`, it got "pulled in" from the newer version of CAPZ manifests.